aws/tags_kinesis.go

```diff
@@ -24,34 +24,55 @@ func setTagsKinesis(conn *kinesis.Kinesis, d *schema.ResourceData) error {
 		// Set tags
 		if len(remove) > 0 {
 			log.Printf("[DEBUG] Removing tags: %#v", remove)
-			k := make([]*string, len(remove), len(remove))
-			for i, t := range remove {
-				k[i] = t.Key
-			}
 
-			_, err := conn.RemoveTagsFromStream(&kinesis.RemoveTagsFromStreamInput{
-				StreamName: aws.String(sn),
-				TagKeys:    k,
-			})
-			if err != nil {
-				return err
+			// Kinesis requires these operations be split into 10 tag batches
+			tagKeysBatchLimit := 10
```

Comment:
Thoughts on pulling this out and making it a constant?

Reply:
That seems fair -- I'll update 👍